scenarios.md

```diff
@@ -199,6 +199,26 @@ A deployment requires a mydb image. The mydb image is routinely updated for secu
 - Multiple signatures, including signatures from multiple sources can be associated with a specific artifact.
 - Original signatures are maintained, even if the artifact is re-tagged.
 
+### Scenario 7: A mirror is compromised
+
+An attacker compromises a server that is hosting a mirror and gains access as an administrator. The attacker is able to alter any files on the mirror and access all keys stored on the mirror.
```

Comment:
We should include a sentence to differentiate a mirror from a copy. The latter would include scenarios with ACME ingesting images from Wabbit Networks and adding their own signatures and policies on the images. I think the key difference between the two is what root key do the clients trust for verifying images. If they verify the upstream root, it acts as a mirror, and if it's a different root, we'd consider that a copy.